test/openAPI/features/wallet_link_status.feature

```diff
@@ -7,29 +7,62 @@ Feature: The endpoint to checks the status of link code.
     And The /linked-authorization/link-status response should contain transactionId property equals provided transactionId
 
   @negative
-  Scenario: Not able to check the status of link code because of the invalid transactionId
+  Scenario: Not able to check the status of link code because of the blank transactionId 
     Given Wants to check the status of link code
-    When Send POST /linked-authorization/link-status request with given X-XSRF-TOKEN header, invalid transactionId, linkCode and requestTime
+    When Send POST /linked-authorization/link-status request with given X-XSRF-TOKEN header, blank transactionId, linkCode and requestTime
     Then Receive a response from the /linked-authorization/link-status endpoint
-    And The /linked-authorization/link-status endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/link-status endpoint response should be returned in a timely manner 25000ms
     And The /linked-authorization/link-status endpoint response should have status 200
     And The /linked-authorization/link-status endpoint response should have content-type: application/json header
     And The /linked-authorization/link-status endpoint response should match json schema with errors
     And The /linked-authorization/link-status response should contain errorCode property equals to "invalid_transaction_id"
 
   @negative
-  Scenario: Not able to check the status of link code because of the invalid linkCode
+  Scenario: Not able to check the status of link code because of the random transactionId
     Given Wants to check the status of link code
-    When Send POST /linked-authorization/link-status request with given X-XSRF-TOKEN header, transactionId, invalid linkCode and requestTime
+    When Send POST /linked-authorization/link-status request with given X-XSRF-TOKEN header, random transactionId, linkCode and requestTime
     Then Receive a response from the /linked-authorization/link-status endpoint
-    And The /linked-authorization/link-status endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/link-status endpoint response should be returned in a timely manner 25000ms
+    And The /linked-authorization/link-status endpoint response should have status 200
+    And The /linked-authorization/link-status endpoint response should have content-type: application/json header
+    And The /linked-authorization/link-status endpoint response should match json schema with errors
+    And The /linked-authorization/link-status response should contain errorCode property equals to "invalid_transaction_id"
+
+  @negative
+  Scenario: Not able to check the status of link code because of the completed transactionId
+    Given Wants to check the status of link code
+    When Send POST /linked-authorization/link-status request with given X-XSRF-TOKEN header, completed transactionId, linkCode and requestTime
```

Comment:
completed transaction id should mean where a transaction has already gone through link-code, authenticate, consent and link auth code